Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.6
+
+RUN mkdir -p /opt
+ADD ./e2e-harness /opt/e2e-harness
+
+RUN mkdir -p /opt/resources
+ADD resources/templates/ /opt/resources/templates
+
+RUN \
+    apk -Uuv add --update \
+      build-base \
```

Comment:
Can we specify particular versions here? Floating versions makes me nervous

Reply:
Indeed, very good catch, done